dd-java-agent/instrumentation/undertow-2.0/undertow-2.0.gradle

```
@@ -0,0 +1,32 @@
+muzzle {
+  pass {
+    group = "io.undertow"
+    module = "undertow-core"
```

```suggestion
    module = "undertow-servlet"
```

Does this resolve the muzzle failure when you run locally?

---

No, I get 
`Task 'muzzle-AssertPass-io.undertow-undertow-core-2.0.0.Final' not found in project ':dd-java-agent:instrumentation:undertow-2.0'.` 
for both